lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -32,11 +32,12 @@ def name
   #
   def commands
     base = {
-      "db_connect"    => "Connect to an existing data service",
-      "db_disconnect" => "Disconnect from the current data service",
-      "db_status"     => "Show the current data service status",
-      "db_save"       => "Save the current data service connection as the default to reconnect on startup",
-      "db_remove"     => "Remove the saved data service entry"
+      "db_connect"       => "Connect to an existing data service",
+      "db_disconnect"    => "Disconnect from the current data service",
+      "db_status"        => "Show the current data service status",
+      "db_save"          => "Save the current data service connection as the default to reconnect on startup",
+      "db_remove"        => "Remove the saved data service entry",
+      "db_clear_default" => "Clears the default data service connection used on startup"
```

Comment:
Thinking aloud; I wonder if it makes sense to just add a flag to `db_disconnect` instead of an additional command? 🤔

Reply:
to `db_disconnect`? no I don't think so, if you just started up the database and ran `db_disconnect` it would disconnect you from the database entirely
what I _did_ have was `db_save` would clear the default before trying to save a new one which did work buuuut it's technically not quite right since it's supposed to only save connections related to the http service so I undid it